week2/project/ecommerce/src/pages/productsList/categories/Categories.jsx

```diff
@@ -0,0 +1,20 @@
+import './Categories.css';
+import Category from './Category'
+
+function Categories ({categories, state, setState, setIsLoading}) {
```

Comment:
`isLoading` is not used
✏️need to modify✏️

Reply:
fixed